lib/ansible/modules/cloud/linode/linode.py

```diff
@@ -16,7 +16,7 @@
 
 ANSIBLE_METADATA = {'status': ['preview'],
                     'supported_by': 'community',
-                    'version': '1.0'}
+                    'version': '1.1'}
```

Comment:
this is a version for the metadata format.  It should not be updated unless the format of the metadata changes.